.github/workflows/Linux-pack.yml

```
@@ -555,7 +563,7 @@ jobs:
           echo "================================"
           echo "VERSION=${last_committed_tag:1}" >> $GITHUB_ENV
       - name: Packaging snap
-        uses: snapcore/action-build@v1
+        uses: snapcore/action-build@v1.2.0
```

```suggestion
        uses: snapcore/action-build@v1
```

`v1` will install latest v1 version ;)